GAMES_LIST.md

```diff
@@ -282,19 +298,22 @@
 | `ut2003`                              | Unreal Tournament 2003                                  |                                             |
 | `ut2004`                              | Unreal Tournament 2004                                  |                                             |
 | `ut3`                                 | Unreal Tournament 3                                     |                                             |
-| `unturned`                            | unturned                                                | [Valve Protocol](#valve)                    |
+| `unturned`                            | Unturned                                                | [Valve Protocol](#valve)                    |
```

Comment:
Apparently, in `games.txt`, the unturned game name is also starting with a small letter, could you also change that? (and also mention capitalizing on the letter in the Changelog as someone might want to know game names changes)

Reply:
Yeah I can add it to the changelog.